### ISSUE TITLE
the last song in the group doesn't seem to function

### ISSUE THREAD

**Issue by hiddendragonXVII** [rufusdumont/kingofahundredstreams#1]
![image](https://cloud.githubusercontent.com/assets/241176/15053268/5cda0166-1334-11e6-9bd3-2161c138f2c9.png)

**Comment by rufusdumont** [comment]
Yeah I noticed that when I skip to some of the later songs it doesn't work. I think it's a Chrome thing because it does work in Safari. Will look into this.
